COMMAND: diff -u /tmp/audit-work/repro/spec.k /tmp/audit-work/repro/spec-vacuity.k
--- /tmp/audit-work/repro/spec.k	2026-07-26 23:18:04.762560090 -0500
+++ /tmp/audit-work/repro/spec-vacuity.k	2026-07-26 23:56:51.486737966 -0500
@@ -1,6 +1,6 @@
 requires "verification.k"
 
-module SPEC
+module SPEC-VACUITY
   imports VERIFICATION
 
   claim [find-max-loop-invariant]:
@@ -47,12 +47,12 @@
          -1 |-> builtinsScope
     </scopes>
 
-  claim [find-max-contract]:
+  claim [find-max-contract-false-result]:
     <k> #loadAll(Module(
          FuncDef("find_max", Params("words"), findMaxFunctionBody)))
          ~> Call(Name("find_max"), list(wordVals(WORDS:WordSeq)))
-      => str(bestWord(findMaxWords(
-           WORDS, .IntSeq, 0)))
+      => str(iCons(119, iCons(114, iCons(111,
+           iCons(110, iCons(103, .IntSeq))))))
     </k>
     <env> 0 </env>
     <scopes>
EXIT_STATUS: 0
